plutus-core/plutus-ir/Language/PlutusIR/Error.hs

```diff
@@ -39,6 +39,9 @@ data TypeErrorExt uni ann =
     deriving (Show, Eq, Generic, NFData)
 makeClassyPrisms ''TypeErrorExt
 
+instance ErrorCode (Language.PlutusIR.Error.TypeErrorExt _a2_acYY _a1_acYX) where
```

Comment:
I don't mind if this is generated with TH, but we should not commit it with nonsense variables.